app/services/data_api/tad_application_export.rb

```diff
@@ -59,7 +59,7 @@ def as_json
         programme_outcome: application_choice.course.description,
         course_name: application_choice.course.name,
         course_code: application_choice.course.code,
-        nctl_subject: concatenate(application_choice.course.subject_codes),
+        nctl_subject: concatenate(application_choice.course.subjects.map(&:code)),
```

Comment:
if we add a method to course called `subject_codes`, that maps those codes from the new `subject` object, we can avoid having to call those changes in every instance, what do you think?

Reply:
Agreed